junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/DefaultLauncher.java

```diff
@@ -250,4 +252,14 @@ private void execute(TestDescriptor engineDescriptor, EngineExecutionListener li
 		}
 	}
 
+	/**
+	 * prints the reason for test cases that are excluded in post discovery filter phase.
+	 * @param root of the all engines.
+	 */
+	private void printFilteredReason(Root root) {
+		root.getExclusionReasonToDescriptorMap().forEach((key, value) -> {
+			String displayNames = value.stream().map(TestDescriptor::getDisplayName).collect(Collectors.joining(", "));
+			logger.debug(() -> String.format("Method names [%s] excluded, reason [%s]", displayNames, key));
```

Comment:
Why this `debug` level? Maybe `info` would make more sense?

Reply:
@ptlanu22 Let’s put two logs here:
- one info log with the number of tests that were not executed, along with the reason
- one debug log with the list of the affected tests (in order to keep the first log concise)